pkg/cri/config/config.go

```diff
@@ -282,6 +282,9 @@ type PluginConfig struct {
 	// of being placed under the hardcoded directory /var/run/netns. Changing this setting requires
 	// that all containers are deleted.
 	NetNSMountsUnderStateDir bool `toml:"netns_mounts_under_state_dir" json:"netnsMountsUnderStateDir"`
+	// ImagePullProgressDeadline is the option that If no pulling progress
+	// is made before this deadline, the image pulling will be cancelled.
+	ImagePullProgressDeadline string `toml:"image_pull_progress_deadline" json:"imagePullProgressDeadline"`
```

Comment:
Would be good to note that the format expected is the go duration format like the comment for `StreamIdleTimeout` calls out above. https://golang.org/pkg/time/#ParseDuration

Reply:
updated with `image_pull_progress_timeout` and used description comment like  `stream_idle_timeout`.